src/App.js

```diff
@@ -1,43 +1,16 @@
-import "./App.css";
-import logo from "./logo.png";
+import "./components/navbar/navbar.css";
+// import {LandingPage} from "./pages/landing-page.jsx"
 
+import { Navbar } from "./components/navbar/navbar.js";
```

Comment:
You can create a separate file where all components are imported.
And then use that file everywhere

Reply:
Wow that I was not aware of